ui/appui-react/src/appui-react/childwindow/InternalChildWindowManager.tsx

```diff
@@ -257,6 +271,25 @@ export class InternalChildWindowManager implements FrameworkChildWindows {
     return outLocation;
   }
 
+  // This is needed due to the differences in how browsers calculate the
+  // inner vs outer width of windows. Calculations are different between
+  // browsers. We are starting with some of the mainly used browsers but can
+  // add more if issues arise with others.
```

Comment:
Can you validate how safari handles this? I suppose Mac users default browser would be safari?

Reply:
Getting the repository set up on my Mac and trying it out